traffic_ops/app/db/seeds.sql

```diff
@@ -97,6 +103,23 @@ FROM public.role
 WHERE "name" in ('operations', 'read-only', 'portal', 'federation', 'steering')
 ON CONFLICT DO NOTHING;
 
+INSERT INTO public.role_capability
+SELECT id, perm FROM public.role
+CROSS JOIN (
+    VALUES ('DNS-SEC:READ'), ('DNS-SEC:DELETE')
+) AS perms(perm)
+WHERE "priv_level" < 20 AND "priv_level" > 8
+    ON CONFLICT DO NOTHING;
```

Comment:
> I increased the check from 0 to 8.. not sure if that is the right way but hopefully it helps.

Additional perms are no longer granted to portal (`6`), but this still grants `DNS-SEC:READ` and `DNS-SEC:DELETE` to a bunch of roles that did not have it before: trouter (`1`), read-only (`4`), steering (`7`), and federation (`8`):

```diff
--- apache.sql	2023-09-05 11:04:30.851468321 -0600
+++ 7764.sql	2023-09-05 11:05:02.572109742 -0600
@@ -3024,6 +3024,14 @@
 6	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
 7	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
 8	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
+1	DNS-SEC:DELETE	A_TIMESTAMP
+4	DNS-SEC:READ	A_TIMESTAMP
+4	DNS-SEC:DELETE	A_TIMESTAMP
+7	DNS-SEC:READ	A_TIMESTAMP
+7	DNS-SEC:DELETE	A_TIMESTAMP
+8	DNS-SEC:READ	A_TIMESTAMP
+8	DNS-SEC:DELETE	A_TIMESTAMP
+1	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
 3	ACME:READ	A_TIMESTAMP
 4	ACME:READ	A_TIMESTAMP
 6	ACME:READ	A_TIMESTAMP
```

Previous convo: https://github.com/apache/trafficcontrol/pull/7764#discussion_r1312398427

Reply:
I think I resolved it this time, 🤞 .